.github/workflows/archive.yml

```diff
@@ -0,0 +1,44 @@
+name: "Archive Issues and Pull Requests"
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,2,4'
```

Comment:
I guess this is because IETF considers all discussions and repo history as normative or such?

Reply:
Maybe we can skip doing this for now, then, until we actually figure out an external storage system. If you decide to do that, though, please file an issue about it so we don't completely forget to do it.